pom.xml

```diff
@@ -290,16 +325,16 @@
         <extensions>true</extensions>
       </plugin>
       <plugin>
-        <!-- OWASP's dependency-check plugin will scan the third party
-             dependencies of this project for known CVEs (security
-             vulnerabilities against them). It will produce a report
-             in target/dependency-check-report.html. To invoke, run
-             'mvn dependency-check:aggregate'. Note that this plugin
-             requires maven 3.1.1 or greater.
-        -->
         <groupId>org.owasp</groupId>
         <artifactId>dependency-check-maven</artifactId>
         <version>${dependency-check-maven.version}</version>
+        <configuration>
```

Comment:
The OWASP plugin should be in a profile so that it is only run with Java11+.

Otherwise site build will  fail with Java 8-10.

Reply:
@stoty not worrying about this; whoever fixes the site build can deal with it. We don't ever build a thirdparty site after all. What's most important is to use java8 for any new release we still want to work with hadoop 3.4.x.